src/components/Fallback/index.tsx

```diff
@@ -13,8 +13,10 @@
 import { Spinner, spinnerSize } from '@patternfly/react-core';
 import React from 'react';
 
-export const fallback = (
+const Fallback = (
```

Comment:
I am proposing to use in this case a function instead of a constant.

Reply:
We can do that later if we need to provide a property to this component.